DESCRIPTION

```diff
@@ -20,16 +20,23 @@ URL: https://github.com/r-lib/gh#readme
 BugReports: https://github.com/r-lib/gh/issues
 Imports:
     cli (>= 2.0.1),
+    curl,
     httr (>= 1.2),
     ini,
-    jsonlite
+    jsonlite,
+    stats,
+    withr
```

Comment:
New Import. If `set_github_pat2()` goes to credentials, can move back to suggests.

Reply:
withr is fine as well